src/panel/app_panel.js

```diff
@@ -55,7 +55,16 @@ export default class AppPanel {
     }
 
     ReactDOM.render(<Menu />, document.querySelector('.react_menu__container'));
-    Telemetry.add(Telemetry.APP_START);
+    Telemetry.add(Telemetry.APP_START, null, null, {
+      'language': window.navigator.language,
+      'preferred_language': window.preferredLanguage,
+      'device_platform': window.navigator.platform,
```

Comment:
I think these two are really borderline in terms of privacy…

Reply:
It's part of the information listed in the ticket. :-/